packages/serverpod/lib/src/database/table.dart

```diff
@@ -11,12 +12,16 @@ abstract class TableRow extends SerializableEntity {
   /// the database.
   int? id;
 
+  /// The table that this row belongs to.
+  Table get table;
+
   /// The name of the table that contains this row.
-  String get tableName;
+  @Deprecated('Use: table.tableName instead, will be removed in 2.0.0.')
+  String get tableName => table.tableName;
 
   /// Will create a serialization of with the fields that are stored in the
   /// database only.
-  //TODO: better name
+  @Deprecated('Will be removed in 2.0.0.')
```

Comment:
This was a better deprecation message than I did. Should have included the version as well. Let's make sure this is the standard for all deprecation messages in the future and maybe do a followup before we release to update all deprecation messages.